logistics_planning_base/views/logistics_schedule_views.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="logistics_schedule_view_tree" model="ir.ui.view">
+        <field name="name">logistics.schedule.tree</field>
+        <field name="model">logistics.schedule</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="picking_id" />
+                <field name="partner_id" />
+                <field name="transport_type" />
+                <field name="input_product_id" />
+                <field name="output_product_id" />
+                <field name="product_uom" />
+                <field name="product_uom_qty" />
+                <field name="scheduled_load_date" />
+                <field name="logistic_price_unit_done" />
+                <field name="carrier_id" />
```

Comment:
Ensure that every field is shown in tree view that is currently available in .xlsx example file